src/main/java/edu/iu/terracotta/utils/TextConstants.java

```diff
@@ -61,5 +61,6 @@ private TextConstants() {
     public static final String GROUP_PENDING = "Error 115: Group_Pending: The student has not been assigned to a group";
     public static final String ASSIGNMENT_LOCKED = "Error 116: The assignment is not open at this moment";
     public static final String LIMIT_OF_SUBMISSIONS_REACHED ="Error 117: You can't answer this assignment again";
+    public static final String MAX_SUBMISSION_ATTEMPTS_REACHED = "Error 118: Max submission attempts already reached";
```

Comment:
Error 118 is already taken (https://github.com/terracotta-education/terracotta/blob/development/src/main/java/edu/iu/terracotta/controller/app/ConditionController.java#L179), can you use error code 150 instead?

(we have a separate issue to get these error codes documented, but see this comment for how to list out existing error codes: https://terracotta-education.atlassian.net/browse/TCOTA-358?focusedCommentId=10277)